.github/workflows/check-code-style.yml

```diff
@@ -68,8 +68,8 @@ jobs:
         sudo apt-get update
         sudo apt-get install -yqq \
             clang-format-${{ env.LLVM_VERSION }} clang-tidy-${{ env.LLVM_VERSION }} \
-            llvm-${{ env.LLVM_VERSION }}-dev libomp-${{ env.LLVM_VERSION }}-dev \
-            libllvmlibc-${{ env.LLVM_VERSION }}-dev \
+            clang-tools-${{ env.LLVM_VERSION }} llvm-${{ env.LLVM_VERSION }}-dev \
```

Comment:
Not sure why the regression has happened now. But clang-tools is a default target according to https://apt.llvm.org/